rental-car-manager/editorextensions/rental-car-manager/src/modal/rentacarmodal.ts

```diff
@@ -227,6 +227,14 @@ export class RentACarModal extends Modal {
         ...carBlockDef,
         boundingBox: carBB,
       });
+
+      if (carBlock) {
+        carBlock.setReferenceKey("ShapeData", {
+          collectionId: this.getOrCreateCarsCollection().id,
+          primaryKey: `"${car.id}"`,
```

Comment:
nit: Can we include a short, quick-to-write inline comment about why the " quotes are here? (something about explicitly indicating that the PK is a string?)